include/api/Metrics.h

```diff
@@ -88,6 +89,16 @@ class Metrics
   IdType lookup(const std::string_view name) const;
   IntType *lookup(IdType id, std::string_view *name = nullptr) const;
 
+  std::optional<IntType *>
+  lookupPtr(const std::string_view name) const
```

Comment:
A const string_view strikes me as odd, but I see it is done elsewhere in the file.